modules/openapi-generator/src/main/resources/kotlin-client/interface_opt_var.mustache

```diff
@@ -5,9 +5,6 @@
     {{#moshi}}
     @Json(name = "{{{vendorExtensions.x-base-name-literal}}}")
     {{/moshi}}
-    {{#gson}}
```

Comment:
@kuFEAR Why did you remove gson from here?

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/5750#issuecomment-626663248 1.

Reply:
Gson @SerializedName target only for method and fields 
@Target({ElementType.FIELD, ElementType.METHOD}) and Kotlin interface fields is not yet determined as field by kotlin compiler, I saw error message there
<img width="601" alt="Screenshot 2020-05-12 at 12 20 06" src="https://user-images.githubusercontent.com/4124981/81666603-ee04bc80-944a-11ea-82b4-de8422029c8a.png">

Reply:
Thanks